### DISCUSSION TITLE
Please make an Unraid app for Pelican Panel

### DISCUSSION THREAD

**Discussion by Northwave** [pelican-dev/panel#766] [Ideas]
Hi, as the title states, I would love to see a ready installable Unraid App in the Unraid Community Applications. Pterodactyl Panel is also available there and I would love to switch, as maybe many others.

thx for your great work so far !

**Comment by notAreYouScared** [comment]
I do have a community apps template that i've made, i can share it with you if you'd like to test it.

**Reply by Northwave** [reply]
Sure ! :)

**Reply by RaionFuyu** [reply]
I would also like to test it because I am in the process of migrating to Pterodactyl and would love to switch directly to Pelican

**Comment by lancepioch** [comment]
Thanks, definitely think is a great idea! Please see here: https://github.com/pelican-dev/panel/issues/769

**Comment by twist3dimages** [comment]
I can't access the web ui

![image](https://github.com/user-attachments/assets/77b81dd9-b6b2-4086-bc71-221900089274)

Nothing in the logs aside from this

![image](https://github.com/user-attachments/assets/087962d6-9be9-4ade-a0e3-3c82501b9cf8)

**Reply by notAreYouScared** [reply]
Yep this is my complaint with it. I've asked for it to be looked into again...

**Reply by RaionFuyu** [reply]
This happens because Caddy is part of the container.
I mounted the following caddyfile to /etc/caddy/caddyfile

```
{
    admin off
    email admin@email.com
}

:80 {
    root * /var/www/html/public
    encode gzip

    php_fastcgi 127.0.0.1:9000
    file_server
}

```
